funding/manager.go

```diff
@@ -1766,11 +1766,15 @@ func (f *Manager) handleFundingAccept(peer lnpeer.Peer,
 
 	pendingChanID := msg.PendingChannelID
 	peerKey := peer.IdentityKey()
+	var peerKeyBytes []byte
+	if peerKey != nil {
+		peerKeyBytes = peerKey.SerializeCompressed()
```

Comment:
nit: given this is only used when error logging, might as well just call there when needed.

Reply:
I didn't want to introduce a potential nil pointer panic just for logging. But I guess we won't really get to this point if the peer key is nil...

Reply:
sgtm!